datasets/librispeech_asr/README.md

```diff
@@ -0,0 +1,164 @@
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** http://www.openslr.org/12
+- **Repository:** [Needs More Information]
+- **Paper:** https://www.danielpovey.com/files/2015_icassp_librispeech.pdf
+- **Leaderboard:** https://paperswithcode.com/sota/speech-recognition-on-librispeech-test-other
+- **Point of Contact:** dpovey@gmail.com
+
+### Dataset Summary
+
+LibriSpeech is a corpus of approximately 1000 hours of 16kHz read English speech, prepared by Vassil Panayotov with the assistance of Daniel Povey. The data is derived from read audiobooks from the LibriVox project, and has been carefully segmented and aligned.
+
+### Supported Tasks and Leaderboards
+
+- `automatic-speech-recognition`, `speaker-identification`: The dataset can be used to train a model for Automatic Speech Recognition (ASR). The model is presented with an audio file and asked to transcribe the audio file to written text. The most common evaluation metric is the word error rate (WER). The task has an active leaderboard which can be found at https://paperswithcode.com/sota/speech-recognition-on-librispeech-test-clean and ranks models based on their WER.
+
+### Languages
+
+The audio is in English. There are two configurations: `clean` and `other`. 
+The speakers in the corpus were ranked according to the WER of the transcripts of a model trained on
+a different dataset, and were divided roughly in the middle,
+with the lower-WER speakers designated as "clean" and the higher WER speakers designated as "other".
+
+## Dataset Structure
+
+### Data Instances
+
+A typical data point comprises the path to the audio file, usually called `file` and its transcription, called `text`. Some additional information about the speaker and the passage which contains the transcription is provided.
+
+{'chapter_id': 141231,
+ 'file': '/home/patrick/.cache/huggingface/datasets/downloads/extracted/b7ded9969e09942ab65313e691e6fc2e12066192ee8527e21d634aca128afbe2/dev_clean/1272/141231/1272-141231-0000.flac',
+ 'id': '1272-141231-0000',
+ 'speaker_id': 1272,
+ 'text': 'A MAN SAID TO THE UNIVERSE SIR I EXIST'}
```

Comment:
````suggestion
```
{'chapter_id': 141231,
 'file': '/home/patrick/.cache/huggingface/datasets/downloads/extracted/b7ded9969e09942ab65313e691e6fc2e12066192ee8527e21d634aca128afbe2/dev_clean/1272/141231/1272-141231-0000.flac',
 'id': '1272-141231-0000',
 'speaker_id': 1272,
 'text': 'A MAN SAID TO THE UNIVERSE SIR I EXIST'}
```
````
use code block